packages/shared/src/components/Feed.tsx

```diff
@@ -250,8 +250,10 @@ export default function Feed<T>({
     onMenuClick,
     onShareMenuClick,
     postMenuIndex,
-    postMenuLocation,
+    postShareMenuIndex,
+    postShareMenuLocation,
     setPostMenuIndex,
+    setPostShareMenuIndex,
```

Comment:
Ideally we want to get rid of more props here.
Rather think some state hook would be better as discussed on slack.

Maybe also take a quick look at `Lean feed` think I had this issue as well and solved it like that.
Basically each component can rather have it's own logic vs prop drilling.

Ref:
https://github.com/dailydotdev/apps/pull/2279